bluesky/plans.py

```diff
@@ -2595,7 +2597,11 @@ def adaptive_core():
         past_I = None
         cur_I = None
         cur_det = {}
-        while next_pos < stop:
+        if stop >= start:
+            direction_sign = 1
+        else:
+            direction_sign = -1
+        while next_pos*direction_sign < stop*direction_sign:
```

Comment:
Minor point: this line needs whitespace around `*` to be PEP-8 compliant.

Reply:
I am kind of confused on how to control whitespace in if or while statement. good catch.